util/flakyt.go

```diff
@@ -46,7 +46,7 @@ func (ft *FlakyT) Cleanup(cls func()) {
 	ft.cls = append([]func(){cls}, ft.cls...)
 }
 
-var numRetries = 5
+var numRetries = 50
```

Comment:
Uhm, I'd lower this to 5 again... or maybe 10. But 50 is too much and will be running forever on a real error.
With the new local-devnet update things should be less flaky.

Reply:
Cool, done... also reverted the `require.Eventually` timeouts to the lower values.